content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +64,185 @@ func TestAccXxx(*testing.T)
-acceptance tests is to use the built in `make` step `testacc`, which explicitly
-sets the `TF_ACC=true` value for you. Example:
+-> Each provider may have more specific requirements and setup recommendations. Check the provider codebase for more details.
 
+#### Terraform CLI Installation Behaviors
+
+The testing framework implements the following Terraform CLI discovery and installation behaviors:
+
+- If the `TF_ACC_TERRAFORM_PATH` environment variable is set, that Terraform CLI binary is used if it exists and is executable. If not found or executable, an error will be returned unless the `TF_ACC_TERRAFORM_VERSION` environment variable is also set.
+- If the `TF_ACC_TERRAFORM_VERSION` environment variable is set, install and use that Terraform CLI version.
+- If both the `TF_ACC_TERRAFORM_PATH` and `TF_ACC_TERRAFORM_VERSION` environment variables are unset, perform a lookup for the Terraform CLI binary based on the operating system `PATH`. If not found, the latest available Terraform CLI binary is installed.
+
+Refer to the [Environment Variables](#environment-variables) section for more details about behaviors and valid configurations.
+
+### Command Line
+
+Run acceptance testing via the command line of any workstation. Use these instructions as the basis for other environments such as continuous integration runners.
+
+The following example will typically execute all available acceptance tests in a provider codebase:
+
+```shell
+TF_ACC=1 go test -v ./...
 ```
-$ make testacc
+
+Some provider codebases also implement a [Makefile](https://www.gnu.org/software/make/manual/make.html#Makefiles) with a `testacc` target, which will set `TF_ACC` and other testing flags automatically.
+
+The following Makefile configuration example:
+
+```make
+testacc:
+  TF_ACC=1 go test -v ./...
 ```
 
-**It’s important to reiterate that acceptance tests create actual cloud resources**,
-possibly incurring expenses which are the responsibility of the user running
-the tests. Creating real infrastructure in
-tests verifies the described behavior of Terraform Plugins in real world use
-cases against the actual APIs, and verifies both local state and remote values
-match. Acceptance tests require a network connection and often require
-credentials to access an account for the given API.
+Enables developers to use the following to run acceptance tests:
+
+```shell
+make testacc
+```
 
-~> Note: When developing or testing Terraform plugins, we highly recommend
-running acceptance tests with an account dedicated to testing. This ensures no
-infrastructure is created or destroyed in error during development or validation
-of any Provider Resources in any environment that cannot be completely and
-safely destroyed.
+### GitHub Actions
+
+If using [GitHub](https://github.com/), run acceptance testing via [GitHub Actions](https://github.com/features/actions). Other continuous integration runners, while not exhaustively documented, are also supported.
```

Comment:
do we have a list of which ones we support?

Reply:
Technically, any. I just didn't want folks to think GitHub Actions was the only way. 😅  We would need to spend some time to figure out configurations for other CI environments like CircleCI, GitLab, etc.

Reply:
hahaha okay fair - I'll let this go